3/index.d.ts

```diff
@@ -3233,6 +3238,11 @@ declare module sequelize {
      */
     having?: WhereOptions;
 
+    /**
+     * Use sub queries (internal)
+     */
+    subQuery?: boolean;
```

Comment:
You can remove the `internal` and instead document that this should only be used if you know for sure the query does no result in a cartesian product